Card/build.gradle

```diff
@@ -12,14 +12,15 @@ android {
         minSdkVersion 21
         targetSdkVersion 30
         versionCode 1
-        versionName "1.0"
+        versionName "0.0.1-testCard"
```

Comment:
Let's pull all these into the top level gradle (I think `versionName` can be the same for all modules)

Reply:
@sarahkoop do you mean that every module is going to have the same `versionName`?

Reply:
Yes in `braintree_android` this field is set at [top level](https://github.com/braintree/braintree_android/blob/master/build.gradle#L35) and corresponds to the version to be released like `1.0.0-SNAPSHOT` or `1.1.0` or 1.0.0-beta`. All modules should be on the same version I believe.

Reply:
@jcnoriega Yeah historically it's been a bit of a compromise. Separate version numbers complicate git tagging and releases in general. We went with a monolith versioning scheme. We mainly use semantic versioning to communicate new features / breaking changes to merchants.